opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -245,36 +245,56 @@ impl<RT: Runtime> PeriodicReaderWorker<RT> {
             Either::Left((res, _)) => {
                 res // return the status of export.
             }
-            Either::Right(_) => {
-                otel_error!(
-                    name: "collect_and_export",
-                    status = "timed_out"
-                );
-                Err(MetricsError::Other("export timed out".into()))
-            }
+            Either::Right(_) => Err(MetricsError::ExportTimeout(
+                "PeriodicReader".into(),
+                self.timeout.as_nanos(),
+            )
+            .into()),
         }
     }
 
     async fn process_message(&mut self, message: Message) -> bool {
         match message {
             Message::Export => {
                 if let Err(err) = self.collect_and_export().await {
-                    global::handle_error(err)
+                    match err {
+                        MetricsError::ExportTimeout(_, _) => {
+                            otel_error!( name: "PeriodicReader.ExportFailed", error  = format!("{:?}", err));
+                        }
+                        MetricsError::ReaderShutdown => {
+                            otel_debug!( name: "PeriodicReader.ReaderShutdown", error = format!("{:?}", err));
```

Comment:
I think we could use `otel_error` for each of these variants as they all seem important. Metrics export is anyway not a hot-path operation so we don't have to worry about flooding of error messages.

Reply:
I thought of doing same way, @cijothomas comment in earlier PRs was to only keep user-actionable one's in the error, and rest in debug - irrespective of not in hot-path.

Reply:
Shutdown failures are propagated to users via Result anyway, so no need of another Error level log.